### PR TITLE
Ground work for raw encoding

### PR DESCRIPTION
**Pull request by vivekrsharma**
This PR lays the ground work for raw encoding. Also has a lot of dependency on transport and needs to be revisited once transport layer is more concrete.

### CODE REVIEW

File: yarpc-encoding-raw/src/main/java/io/yarpc/encoding/raw/UnaryRawHandler.java [hunk 1]
```diff
@@ -21,14 +21,21 @@
...
 
-public class RawInboundHandler extends AbstractInboundHandler {
-    private static final Logger LOGGER = LoggerFactory.getLogger(RawInboundHandler.class);
+import io.yarpc.core.transport.TransportException;
+import io.yarpc.core.transport.TransportRequest;
+import io.yarpc.core.transport.TransportResponseWriter;
+import io.yarpc.core.transport.UnaryHandler;
+
+public class UnaryRawHandler implements UnaryHandler {
+  private static final Logger LOGGER = LoggerFactory.getLogger(UnaryRawHandler.class);
```
Comment (abhinav) [inline yarpc-encoding-raw/src/main/java/io/yarpc/encoding/raw/UnaryRawHandler.java]: Don't declare things that aren't being used yet

**Comment by breerly** [comment]
@abhinav is it crazy talk to somehow commit/share the IDE files responsible for controlling these settings? I haven't used JetBrains in awhile, but I seemed to remember something like that.

**Comment by abhinav** [comment]
No, it's not crazy talk. IIRC there was such a feature, but I've been holding
off on setting up that side of things. Alternatively, we could do what Vivek
suggested: Adopt Google's Java Style and use [google-java-format] to format
everything for us.

  [google-java-format]: https://github.com/google/google-java-format

File: yarpc-encoding-raw/src/main/java/io/yarpc/encoding/raw/UnaryRawHandler.java (hunk 1 above)
Comment (vivekrsharma) [inline yarpc-encoding-raw/src/main/java/io/yarpc/encoding/raw/UnaryRawHandler.java]: Done.

File: yarpc-encoding-raw/src/main/java/io/yarpc/encoding/raw/OneWayRawHandler.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
+package io.yarpc.encoding.raw;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
```
Comment (abhinav) [inline yarpc-encoding-raw/src/main/java/io/yarpc/encoding/raw/OneWayRawHandler.java]: Unused imports

File: yarpc-encoding-raw/src/main/java/io/yarpc/encoding/raw/OneWayRawHandler.java [hunk 3]
```diff
@@ -0,0 +1,17 @@
+package io.yarpc.encoding.raw;
```
Comment (abhinav) [inline yarpc-encoding-raw/src/main/java/io/yarpc/encoding/raw/OneWayRawHandler.java]: Missing license header

File: yarpc-encoding-raw/src/main/java/io/yarpc/encoding/raw/UnaryRawHandler.java [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT
+ * NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+ * NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+ * DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+package io.yarpc.encoding.raw;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
```
Comment (abhinav) [inline yarpc-encoding-raw/src/main/java/io/yarpc/encoding/raw/UnaryRawHandler.java]: Unused imports

**Pull request merged**